rust/agama-dbus-server/src/web/docs.rs

```diff
@@ -3,7 +3,11 @@ use utoipa::OpenApi;
 #[derive(OpenApi)]
 #[openapi(
     info(description = "Agama web API description"),
-    paths(super::http::ping),
-    components(schemas(super::http::PingResponse))
+    paths(super::http::ping, crate::l10n::web::locales),
```

Comment:
This shows in the `agama-web-server openapi` output as `/locales` path whereas the actual path is `/l10n/locales`